### PR TITLE
Updated README and added intentions section

### LINKED ISSUE #65: Add intentions documentation
## What is the issue?
No documentation about intentions.

## Suggestion(s)/solution(s) [Optional]
Add intentions documentation in README.md

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by dangernil**
## Closes/fixes/resolves issue(s)?
Closes #65 

## What was added/changed/fixed?
Updated contents in README and added intentions section

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,66 @@
...
+## Content
+1. [Terraform-nomad-minio](#terraform-nomad-minio)
+2. [Compatibility](#compatibility)
+3. [Usage](#usage)
+    1. [Providers](#providers)
+    2. [Intentions](#intentions)
+4. [Requirements](#requirements)
+    1. [Required modules](#required-modules)
+    2. [Required software](#required-software)
+    3. [Other](#other)
```
Comment (claesgill) [inline README.md]: I would prefer having requirements before Usage. What do you think?

**Review by pdmthorsrud — DISMISSED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
+4. [Usage](#usage)
+    1. [Providers](#providers)
+    2. [Intentions](#intentions)
 5. [Inputs](#inputs)
 6. [Outputs](#outputs)
     1. [Example](#example)
 7. [Vault secrets](#vault-secrets)
 8. [Volumes](#volumes)
-9. [Verifyin setup](#verifying-setup)
+9. [Verifying setup](#verifying-setup)
```
Comment (claesgill) [inline README.md]: Didn't see before now, but the order should be:
```
6. [Outputs](#outputs)
7. [Vault secrets](#vault-secrets)
8. [Volumes](#volumes)
9. [Example](#example)
    1. [Verifyin setup](#verifying-setup)
```
*Remember to move the actual sections as well*

File: README.md [hunk 3]
```diff
@@ -100,6 +98,21 @@ In the examples, intentions are created in the Ansible playboook [00_create_inte
...
+vault kv get -field='access_key' secret/minio
+
+# get secret_key
+vault kv get -field='secret_key' secret/minio
+```
+## Volumes
+We are using [host volume](https://www.nomadproject.io/docs/job-specification/volume) to store minio data.
+Minio data will now be available in the `persistence/minio` folder.
 
 ### Example
```
Comment (claesgill) [inline README.md]: Need to remove one `#`

**Review by claesgill — APPROVED** [review]
LGTM!

**Review by pdmthorsrud — APPROVED** [review]

**Pull request merged**